src/components/employeeDetails.vue

```diff
@@ -41,11 +37,11 @@ export default {
   data() {
     return {
       currentEmployee: '',
-      message: '',
-      employeeName: '',
-      position: '',
-      dateOfHire: '',
-      status: ''
+      message: ''
+      // employeeName: '',
```

Comment:
What are these for and do we need them?

Reply:
They were originally added for validation, but not needed at the end, they will be removed.